packages/dom/src/serialize-cssom.js

```diff
@@ -18,6 +18,25 @@ function styleSheetsMatch(sheetA, sheetB) {
   return true;
 }
 
+function styleSheetFromNode(node) {
+  /* istanbul ignore if: sanity check */
+  if (node.sheet) return node.sheet;
+  /* istanbul ignore if: sanity check */
+  if (node.constructor.name !== 'HTMLStyleElement') return;
```

Comment:
```suggestion
```

we'd ideally want to break here, otherwise stuff ahead of this will break and we won't have a clue.

Reply:
* keep this code and debug with `youtube.com` and understand what value `node.constructor.name` has in that case.
  * we can include that case as well.

Reply:
It was happening for `HTMLSVGStyleElement`, which is just a style tag. Its same to remove the condition

Reply:
Nice